kickstarts/partials/main/repos.ks.erb

```diff
@@ -12,3 +12,4 @@ repo --name=google-cloud-compute --baseurl=https://packages.cloud.google.com/yum
 # Please also add to "post install repos" post/repos partial
 repo --name=ansible-runner --baseurl=https://releases.ansible.com/ansible-runner/rpm/epel-8-x86_64/
 repo --name=manageiq-ManageIQ-Master --baseurl=https://copr-be.cloud.fedoraproject.org/results/manageiq/ManageIQ-Master/epel-8-x86_64/
+repo --name=simaishi-test --baseurl=https://copr-be.cloud.fedoraproject.org/results/simaishi/test/epel-8-x86_64/
```

Comment:
should `simaishi-test` be a more generic name?

Reply:
This is temporary for testing purpose. https://github.com/ManageIQ/manageiq-appliance-build/pull/415/commits/a40faff8dcb961cd7b9048f8ffca835a6b318580 will be removed before merge.

Reply:
Removed the commit.